### PR TITLE
T dh 602 Bulk Water Critical update

### PR DESCRIPTION
**Pull request by byarsawich**

### CODE REVIEW

File: src/classes/BulkWaterHandler.cls [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+				[SELECT Id
+				FROM BusinessHours
+				WHERE Name = :BulkWaterBH];
+
+		 //commented out until I can update the base package
+		//Id woWTRecordType = base_toc.DescribeUtils.getRecordTypeIdByDevName('WorkType', 'General_Work_Type');
+		Id woWTRecordType = base_toc.DescribeUtils.getRecordTypeId('WorkType', 'General Work Type');
+		String readName = Test.isRunningTest() ? 'TEST BULK READ' : 'Bulk Meter Read';
+		WorkType readWt =
+				[SELECT Id, Division__c, Job_Type__c, Name, Work_Order_Record_Type__c, EstimatedDuration,
```
Comment (Fitchburgh) [inline src/classes/BulkWaterHandler.cls]: Is there any way to get these work types from one query/is it worth making the effort to do so?

File: src/classes/BulkWaterHandler.cls-meta.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>41.0</apiVersion>
```
Comment (Fitchburgh) [inline src/classes/BulkWaterHandler.cls-meta.xml]: 43(44?) unless 41 is req

File: src/classes/Test_BulkWaterHandler.cls-meta.xml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>41.0</apiVersion>
```
Comment (Fitchburgh) [inline src/classes/Test_BulkWaterHandler.cls-meta.xml]: 43(44?) unless 41 is req

File: src/classes/BulkWaterHandler.cls (hunk 1 above)
Comment (byarsawich) [reply src/classes/BulkWaterHandler.cls]: I did, but was in the branch for 604 so the changes will be there

File: src/classes/Test_BulkWaterHandler.cls-meta.xml (hunk 3 above)
Comment (byarsawich) [reply src/classes/Test_BulkWaterHandler.cls-meta.xml]: changes are in 604

**Pull request merged**